make-options.json

```diff
@@ -3,6 +3,7 @@
         "ANT",
         "Gulp",
         "NuGetInstaller",
+        "PublishSymbols",
```

Comment:
I figured it out.  I ignore common .d.ts because we were building in place and they were generated but it also led to me not adding.  Now that I changed it to build to an output dir, not an issue.  I'm about to checkin then you can rebase.